spec/01-unit/01-db/01-schema/11-declarative_config/02-process_auto_fields_spec.lua

```diff
@@ -169,12 +169,16 @@ describe("declarative config: process_auto_fields", function()
               config = {
                 http_endpoint = "https://example.com",
                 content_type = "application/json",
-                flush_timeout = 2,
                 keepalive = 60000,
                 method = "POST",
-                queue_size = 1,
-                retry_count = 10,
                 timeout = 10000,
+                queue = {
+                  batch_max_size = 1,
+                  capacity = 10000,
+                  max_delay = 1,
+                  max_retry_delay = 60,
+                  max_retry_time = 60,
+                },
```

Comment:
Do we have some written example what this means. I can see all settings are kinda like `max` settings, but what does it mean internally.

So there can be:
1. 10.000 entries waiting in a queue (at maximum before it starts to error out?)
2. we wait maximum of 1 seconds before we start calling handler?
3. we call handler with a single entry at the time?
4. if handler errors we will retry this single item after waiting at maximum of 60 seconds? And because max_retry_time is set to 60 we will retry a single time? Or because these settings are max_ does it mean they can also be lower and we retry more than once?

I find these settings quite hard to understand just by reading their names.

Reply:
There will be additional user-facing documentation that describes what the parameters do.